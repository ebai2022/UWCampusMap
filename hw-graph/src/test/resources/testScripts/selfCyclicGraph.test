# Test a cyclic graph

# Create a graph
CreateGraph A

# Add node
AddNode A e1

# Add edges
AddEdge A e1 e1 circle

# Check if the graph is cyclic
ListChildren A e1