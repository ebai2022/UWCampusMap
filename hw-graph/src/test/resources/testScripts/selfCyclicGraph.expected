# Test a cyclic graph

# Create a graph
created graph A

# Add node
added node e1 to A

# Add edges
added edge circle from e1 to e1 in A

# Check if the graph is cyclic
the children of e1 in A are: e1(circle)